lesson02/assignment/css/_components.content.scss

```diff
@@ -2,16 +2,158 @@ html {
     box-sizing: border-box;
 }
 
-*,
-*:after,
-*:before {
-    box-sizing: inherit;
+// Creating the flex container as a row, using reusable variables in case we were to add to this single employee card later
+.container {
+    // padding: 2rem;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    width: 60%;
```

Comment:
I would give a component with a known dimension (here we know the size of the card's width) an explicit maximum width. So you could just swap that to `width: 100%;` and `max-width: XXpx;` so it still breaks down as expected but doesn't grow too large.-

Reply:
I thought we weren't supposed to use pixels but should use rems and % where possible for responsive? I used 16 px to calculate rems for this. I'm happy to change it to pixels but wondered about previous teachings.

Reply:
Actually, I think I get it. The width 100% takes care of the fact that we've specified a max-width.

I think.

Reply:
You want percentages for proportions, but for something like a container or component's max-width it's normal to use pixels.